Frameworks/Foundation/NSDictionary.mm

```diff
@@ -99,8 +99,11 @@ - (NSArray*)allObjects {
         return nil;
     }
 
-    StrongId<NSArray> keyArray = [_keyEnumerator allObjects];
-    StrongId<NSMutableArray> toRet = [NSMutableArray new];
+    StrongId<NSArray> keyArray;
+    keyArray.attach([_keyEnumerator allObjects]);
```

Comment:
StrongId isn't the right choice for these, since autorelease takes care of it delightfully well.

---
In reply to: [73979408](https://github.com/Microsoft/WinObjC/pull/851#discussion_r73979408) [](ancestors = 73979408)